lib/tasks/vmodl_helper.rb

```diff
@@ -87,34 +87,22 @@ def verify!
   def generate!
     wsdl_types_by_name.each_value do |type|
       type_name  = type.name.name
-      vmodl_data = @vmodl[type_name]
-
-      if vmodl_data.nil?
-        base_class           = wsdl_types_by_name[type.complexcontent.extension.base.name]
-        inherited_properties = base_class.elements.map { |element| element.name.name }
-        properties           = type.elements.reject { |e| inherited_properties.include?(e.name.name) }
-
-        vmodl_data = {
-          'kind'      => 'data',
-          'props'     => properties.map do |element|
-            {
-              'name'           => element.name.name,
-              'is-optional'    => element.minoccurs == 0,
-              'is-array'       => element.maxoccurs != 1,
-              'version-id-ref' => nil,
-              'wsdl_type'      => wsdl_to_vmodl_type(element.type)
-            }
-          end,
```

Comment:
Since the only place we were processing the list of properties in the wsdl (aka the most up to date list) was here which was a completely new object, we were skipping over new properties added to existing objects.